### DISCUSSION TITLE
Top 10 levels

### DISCUSSION THREAD

**Discussion by kironia** [eltikia/openroguelike#7] [Q&A] [answered]
What would be your top 10 favourite levels?

**Comment by Stagyrite** [comment, marked as answer]
Here's my list.

- A blank level with a player and a dog only. This is how [the 3DS port](https://github.com/Stagyrite/openroguelike/releases/tag/v0.1 "Release 3DS Homebrew · Stagyrite/openroguelike") starts.
- The blank level but with a kitten instead of the dog.
- A town level from [a separate discussion](https://github.com/eltikia/openroguelike/discussions/5#discussioncomment-9141602 "Custom level · eltikia/openroguelike · Discussion #5").
- [A maze](https://github.com/Stagyrite/openroguelike/blob/master/misc/pathfinding-demo-03 "openroguelike/misc/pathfinding-demo-03 at master · Stagyrite/openroguelike") generated with [jaldhar/Maze](https://github.com/jaldhar/Maze "jaldhar/Maze: Maze generation algorithm demo").
- [The Hall of the Goblin King](https://github.com/Stagyrite/openroguelike/blob/master/misc/hall "openroguelike/misc/hall at master · Stagyrite/openroguelike")
- A level from the 3DS port called [dsband](https://github.com/Stagyrite/openroguelike/tree/dsband "Stagyrite/openroguelike at dsband").
- An entry level that has [the entry to the Goblins Caves](https://github.com/Stagyrite/openroguelike/blob/master/misc/entry "openroguelike/misc/entry at master · Stagyrite/openroguelike").
- A level from a [decgraphics ](https://github.com/tleguern/openroguelike/blob/decgraphics/misc/level0 "openroguelike/misc/level0 at decgraphics · tleguern/openroguelike") branch.
- A level with killable goblins.
- A level that consists of non-connected caves. Below is an example generated with [OpenRoguelike.java](https://gist.github.com/Stagyrite/921e5e48465e92ee0fe829b4e32b2c05 "a level compiler inspired by openroguelike").
 
```
================================================================================
======     ===   ===============================================================
=====             =======  =====================================================
=====              =====       ============  =============   ===================
=====                ==         ==========    ========        ==================
======                           =========     ======          =================
=======                          =========       ====           ============  ==
========                          ===             ====           =========     =
==========                                         ====          ========      =
===========                                         === <       =========      =
====  ======                                         ===       ==========      =
===      ====                                        ====      ==========      =
===       ===                                         ===     ====  =====      =
==        ===                                         ===     ===     ==       =
==         ==                                         ==      ===              =
==             @                                     ===     ===               =
==              d                                   ====     ==               ==
===                                                =====     ==         ==   ===
====        =====   ==          >     ==  =====   =======   ====  ==  ==========
================================================================================
```
